test/findBin.spec.js

```diff
@@ -8,25 +8,25 @@ describe('findBin', () => {
   it('should favor `npm run` command if exists in both package.json and .bin/', () => {
     const { bin, args } = findBin('my-linter', { 'my-linter': 'my-linter' })
     expect(bin).toEqual('npm')
-    expect(args).toEqual(['run', '--silent', 'my-linter'])
+    expect(args).toEqual(['run', '--silent', 'my-linter', '--'])
```

Comment:
Can we only add this if there are arguments?

Reply:
I thought of that.. we need it cause we pass file names later on.